dynamiq/nodes/agents/orchestrators/adaptive_manager.py

```diff
@@ -7,9 +7,10 @@
 2. Break down the task into subtasks
 3. Delegate subtasks to appropriate specialized agents
 4. Synthesize results from specialized agents
-5. Request clarification from the user when needed
+5. Request clarification from the user when needed (If appropriate tools are available)
```

Comment:
Don't assume that an LLM can recognize the "appropriate tool." It's better to rely on "human feedback" for sure.